File: client-side/js/Model/City/City.js

function City(code, name, country) {
    this.City(code, name, country);
}

City.prototype = {
    City: function (code, name, country) {
        this.code = code;
        this.name = name;
        this.country = country;
    },
    completo: function (sep) { return this.code + sep + this.name; }
}

File: Airline/src/java/una/airline/domain/City.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package una.airline.domain;

/**
 *
 * @author michaelcw02
 */
public class City {

    private String code;
    private String name;
    private String country;

    public City() {
    }
	
    public City(String code, String name, String country) {
        this.code = code;
        this.name = name;
        this.country = country;
    }
   
    public String getCode() {
        return this.code;
    }
    
    public void setCode(String code) {
        this.code = code;
    }
    public String getName() {
        return this.name;
    }
    
    public void setName(String name) {
        this.name = name;
    }
    public String getCountry() {
        return this.country;
    }
    
    public void setCountry(String country) {
        this.country = country;
    }

}